### PR TITLE
Add episode 01 (Assigning values to variables).

### PR DESCRIPTION
**Pull request by mkcor**
Dear @maikia,

I have adapted the episode on [Python Fundamentals](https://swcarpentry.github.io/python-novice-inflammation/01-intro/index.html) with some slight inspiration from [Variables and Assignment](http://swcarpentry.github.io/python-novice-gapminder/02-variables/index.html).

Line 155, I'm referring to your episode about functions; maybe we could add a link to it afterwards, when you submit it.

Cheers,
Marianne

### CODE REVIEW

File: _episodes/01-variables.md [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+patient_id = 'inflam_' + patient_id
+~~~
+{: .language-python}
+
+Variables must be created before they are used.
+
+## Using Built-in Functions
+
+To carry out common tasks with data and variables in Python,
+the language provides us with several built-in functions.
```
Comment (maikia) [inline _episodes/01-variables.md:122]: maybe worth adding a short sentence on what `built-in` means?

File: _episodes/01-variables.md [hunk 2]
```diff
@@ -0,0 +1,356 @@
...
+{: .language-python}
+~~~
+132.66
+inflam_001
+~~~
+{: .output}
+
+When we want to make use of a function, which is referred to as _calling_ a
+function, we follow its name by parentheses. The parentheses are important:
+If you leave them off, the function doesn't actually run.
```
Comment (maikia) [inline _episodes/01-variables.md]: worth showing them what happens and that it's not an error, though function is not called

File: _episodes/01-variables.md [hunk 3]
```diff
@@ -0,0 +1,356 @@
...
+~~~
+{: .output}
+
+We will learn more about how functions work and how to create our own in a
+later episode.
+
+We can display multiple things at once using only one `print` call:
+
+~~~
+print(patient_id, 'weight in kilograms:', weight_kg)
```
Comment (maikia) [inline _episodes/01-variables.md:174]: just a suggestion: how about also showing them how to print f-string? it's not much more complex than what you already show and very useful also for other things?

File: _episodes/01-variables.md [hunk 4]
```diff
@@ -0,0 +1,356 @@
+---
+title: Assigning values to variables
+teaching: 10
```
Comment (maikia) [inline _episodes/01-variables.md:3]: this renders to a very wide table, perhaps we could put it row under row? instead as columns?

File: _episodes/01-variables.md [hunk 5]
```diff
@@ -0,0 +1,356 @@
...
+weight_kg = 65.0
+print('weight in kilograms is now:', weight_kg)
+~~~
+{: .language-python}
+~~~
+weight in kilograms is now: 65.0
+~~~
+{: .output}
+
+> ## Variables as Sticky Notes
```
Comment (maikia) [inline _episodes/01-variables.md:265]: why to change the formatting from here on?

**Review by maikia — APPROVED** [review]
Thanks @mkcor , LGTM
Few minor comments but feel free to ignore them if you don't agree

File: _episodes/01-variables.md (hunk 4 above)
Comment (mkcor) [reply _episodes/01-variables.md:3]: Right; would you happen to know the syntax for this? :no_mouth:

File: _episodes/01-variables.md (hunk 1 above)
Comment (mkcor) [reply _episodes/01-variables.md:122]: I think that the word 'built-in' and the sentence 'the language provides us with several built-in functions' are self-descriptive (I mean, in plain English). But I'll add a note (to my personal 'instructor notes') to emphasize this when I speak (something along the lines of "they are *built into* the language itself, which means they are readily available for use, as opposed to functions which belong to specific collections, which need importing -- we'll touch on this later in the lesson"). What do you think?

File: _episodes/01-variables.md (hunk 3 above)
Comment (mkcor) [reply _episodes/01-variables.md:174]: I have thought about it, I hesitated, and I thought I shouldn't add any cognitive load... But, since you're suggesting, I'm happy to add it, specifying that we're seeing this just in passing (just so they've heard of it). Thanks for the suggestion!

Comment (mkcor) [reply _episodes/01-variables.md:174]: Done at 8d9dbaa043.

File: _episodes/01-variables.md (hunk 5 above)
Comment (mkcor) [reply _episodes/01-variables.md:265]: Oh, it comes from: https://swcarpentry.github.io/python-novice-inflammation/01-intro/index.html#variables-as-sticky-notes "Variables as Sticky Notes" is a 'callout' section and the rest are challenges (see https://swcarpentry.github.io/python-novice-inflammation/01-intro/index.html#check-your-understanding and so on).

Basically, I left the original formatting... I think it doesn't hurt, but I should probably copy SWC's CSS files for this render as expected.

File: _episodes/01-variables.md (hunk 2 above)
Comment (mkcor) [reply _episodes/01-variables.md]: Done at 38d72de9c35.

**Comment by mkcor** [comment]
@maikia thank you so much for the attentive review! It really benefits the content to have this second pair of eyes. I have addressed your comments; if nothing else comes up, I will merge tomorrow. Thanks again!

File: _episodes/01-variables.md (hunk 4 above)
Comment (maikia) [reply _episodes/01-variables.md:3]: lol :-) not at all. I have actually never wrote my tutorial in .md so we will have a laugh when you review mine ;-)

Feel free to ignore, we will use the documentation (and not github display) for it anyway in which case it's just used for template and does not display as a table

File: _episodes/01-variables.md (hunk 5 above)
Comment (maikia) [reply _episodes/01-variables.md:265]: great. For now it just displays as a shifted paragraph so I was confused :-)

**Review by maikia — APPROVED** [review]
LGTM, thanks @mkcor

File: _episodes/01-variables.md (hunk 4 above)
Comment (mkcor) [reply _episodes/01-variables.md:3]: Haha, ok, no worries!

**Comment by mkcor** [comment]
Thanks again, @maikia! I'll merge now then.
